c7n/mu.py

```diff
@@ -105,9 +106,23 @@ def create(self):
                     self.src_filter(root, dirs, files)
                 files = self.filter_files(files)
                 for f in files:
-                    self._zip_file.write(
-                        os.path.join(root, f),
-                        os.path.join(arc_prefix, f))
+                    f_path = os.path.join(root, f)
+                    info = zipfile.ZipInfo(os.path.join(arc_prefix, f))
```

Comment:
rather than copying and pasting the lines, preference would be to extract out to a utility function that's invoked from both sites.